.evergreen/config.yml

```diff
@@ -357,7 +357,10 @@ functions:
         script: |
           set -o xtrace
           ${PREPARE_SHELL}
-          PYTHON_BINARY=${PYTHON_BINARY} bash ${PROJECT_DIRECTORY}/.evergreen/run-mockupdb-tests.sh
+
+          cd ${PROJECT_DIRECTORY}
```

Comment:
I think this `cd` can be removed because we're already in the `working_dir: "src"`.